src/vellum/workflows/types/code_execution_node_wrappers.py

```diff
@@ -71,7 +71,7 @@ def __getattr__(self, attr):
                 # several values as VellumValue objects, we use the "value" key to return itself
                 return self
 
-            raise AttributeError(f"dict has no key: '{attr}'")
+            return None
```

Comment:
[q] does old jinja return None or their own empty singleton? (the analog to our `undefined`)